datasets/san_francisco_bikeshare_trips/bikeshare_trips/pipeline.yaml

```diff
@@ -80,7 +80,8 @@ dag:
         bucket: "{{ var.value.composer_bucket }}"
         source_objects: ["data/san_francisco_bikeshare_trips/bikeshare_trips/data_output.csv"]
         source_format: "CSV"
-        destination_project_dataset_table: "san_francisco_bikeshare_trips.bikeshare_trips"
+        destination_project_dataset_table: "{{ var.json.san_francisco_bikeshare_trips.container_registry.bikeshare_trips_destination_table }}"
```

Comment:
Please change to `san_francisco_bikeshare` as well

Reply:
@adlersantos Please see my last comment